update-integration-configs/task.yml

```diff
@@ -49,3 +49,11 @@ params:
   GIT_COMMIT_USERNAME: "CI Bot"
   # - Optional
   # - You may choose the git committer username and email address by setting these
+
+  PRIVATE_DOCKER_REGISTRY_IMAGE:
+  PRIVATE_DOCKER_REGISTRY_USERNAME:
+  PRIVATE_DOCKER_REGISTRY_PASSWORD:
+  # - Optional
+  # - You may add private docker registry information to the integration-configs
+  # - Specifying PRIVATE_DOCKER_REGISTRY_IMAGE will automatically assume that the
```

Comment:
Could you please add logic for this? Something like this within the `if [ -n "${PRIVATE_DOCKER_REGISTRY_IMAGE}" ]; then` block :

```
if [[ -n "${PRIVATE_DOCKER_REGISTRY_USERNAME}" ]] &&
   [[ -n "${PRIVATE_DOCKER_REGISTRY_PASSWORD}" ]] ; then
  # do the good stuff
else
  echo "All 3 private docker registry keys must be provided. (PRIVATE_DOCKER_REGISTRY_IMAGE, PRIVATE_DOCKER_REGISTRY_USERNAME, and PRIVATE_DOCKER_REGISTRY_PASSWORD)"
  exit 1
fi
```